src/edu.umn.cs.melt.copper.compiletime/src/main/java/edu/umn/cs/melt/copper/main/ParserCompilerParameters.java

```diff
@@ -333,6 +344,26 @@ public boolean isWarnUselessNTs()
 		return isWarnUselessNTs;
 	}
 
+	public boolean isColorCounterexample()
+	{
+		return colorCounterexample;
+	}
+
+	public File getGraphizDotOutput()
+	{
+		return graphizDotOutput;
+	}
+
+	public void setColorCounterexample(boolean colorCounterexample)
+	{
+		this.colorCounterexample = colorCounterexample;
+	}
+
+	public void setGraphizDotOutput(File graphizDotOutput)
```

Comment:
We should probably add a flag in Silver to enable this - or make it happen with `--copperdump`.  

Also I agree with August's naming comment - should probably just be `dotOutput`, unless you are doing something that is non-standard dot and specific to GraphViz.

Reply:
Is there a particular preference on which of those should be done? I could add it to copperdump without too much issue, but I haven't done so yet.

Reply:
How much time does it add for a large parser, e.g. ableC?  I'd say add it to `--copperdump` if it's no more than a few seconds, otherwise make it a seperate flag.

Reply:
It's pretty fast, so I'll just add it to copperdump. I did find another (more or less unrelated) bug (that I thought I squashed...) when testing that again, so I'll fix that as well when I add it. Not sure how long that will take, but hopefully I will be done by the end of the week.

Reply:
Alright, I looked at it and realized that I think it would require a non-trivial refactor to add to copperdump, so it would be best to add the flag to silver if it's not too much of a pain.